modules/CMakeLists.txt

```diff
@@ -41,3 +41,4 @@ add_subdirectory(systemd-journal)
 add_subdirectory(system-source)
 add_subdirectory(tagsparser)
 add_subdirectory(xml)
+add_subdirectory(openbsd)
```

Comment:
Note: We tried to follow here alphabetical ordering.

Reply:
done. I moved the new module in alphabetical order.